web-backend/src/main/java/com/statusneo/vms/model/Visitor.java

```diff
@@ -103,6 +103,14 @@ public class Visitor {
     @Column(name = "laptop_number")
     private String laptop;
 
+    @Column(name = "valid_govt_id")
+    @Pattern(regexp = "^[0-9]{12}$|^[A-Z]{5}[0-9]{4}[A-Z]{1}$", message = "Invalid Aadhar or PAN format")
```

Comment:
Let's not over design this.
If we have to validate this, change the message to 
"Please enter valid characters"
The chance of people carrying and entering PAN is low.
For most people it would be their DL.